client/src/layers/Roadmap/index.tsx

```diff
@@ -18,13 +21,23 @@ const Roadmap = ({ projectId }: RoadmapProps) => {
   const [inputVisible, setInputVisible] = React.useState(false);
   const [nowDragging, setNowDragging] = React.useState({ id: 0, over: 0 });
   const epicsOnProject = useEpicState();
+  const user = useRecoilValue(userAtom);
```

Comment:
```suggestion
  const userState = useRecoilValue(userAtom);
```
로 쓰시는 건 어떨까요...ㅎㅎ 다른곳에서도 이렇게 쓰고 있어서요,,,